wide-github.css

```diff
@@ -34,15 +34,20 @@ header .container-lg {
 .repository-content .timeline-new-comment { /* New Issue / issue comment form */
   max-width: 100% !important;
 }
-.repository-content .inline-comments .comment-holder { /* Diff / code comments */
-  max-width: 960px !important;
-}
-.repository-content .inline-comments .inline-comment-form-container {
-  max-width: 960px !important;
-}
-.repository-content .inline-comments .inline-comment-form {
-  max-width: 960px !important;
-}
+".repository-content .inline-comments .comment-holder {" + // Diff / code comments
+  "max-width: inherit !important;" +
+"}" +
+".repository-content .inline-comments .inline-comment-form-container {" +
+  "max-width: inherit !important;" +
+"}" +
+".repository-content .inline-comments .inline-comment-form {" +
+  "max-width: inherit !important;" +
+"}" +
+
+".repository-content #all_commit_comments .commit-comments-heading," +
+".repository-content #all_commit_comments .comment-holder {" +
+  "max-width: inherit !important;" +
```

Comment:
Resample for #34 *(click View changes button if looking in the PR)* *(sorry for dupes GH is picky about showing in PR)*